webargs/fields.py

```diff
@@ -36,3 +36,19 @@ def __init__(self, nested, *args, **kwargs):
         if isinstance(nested, dict):
             nested = argmap2schema(nested)
         super(Nested, self).__init__(nested, *args, **kwargs)
+
+class DelimitedList(ma.fields.List):
+
+    delimiter = ','
+
+    def __init__(self, cls_or_instance, delimiter=None, **kwargs):
+        self.delimiter = delimiter or self.delimiter
+        super(DelimitedList, self).__init__(cls_or_instance, **kwargs)
+
+    def _serialize(self, value, attr, obj):
+        ret = super(DelimitedList, self)._serialize(value, attr, obj)
+        return self.delimiter.join(ret)
```

Comment:
Sounds good. Maybe this isn't right, but I expect to be able to round-trip serializing and deserializing--it seems wrong if a serializer can't load what it dumps. Should `_deserialize` accept a list or a string for round-trip friendliness?